README.md

```diff
@@ -18,9 +18,9 @@ To build on Linux and create a Linux binary, use Docker. For example:
 
 The above command will run the build using the latest Swift 5.1 Docker image, utilizing bind mounts to the sources on your Mac. Apple publishes Docker images to Docker Hub.
 
-By default, SwiftPM will build a debug version of the application. Note that debug versions are not suitable for running in production as they are significantly slower. To build a release version of your app, with debug symbols, run `swift build -c release -Xswiftc -g`
+By default, SwiftPM will build a debug version of the application. Note that debug versions are not suitable for running in production as they are significantly slower. To build a release version of your app, run `swift build -c release`.
```

Comment:
should we make a note of using -g in pre 5.2 versions or you feel it will add too much noise? (we have such conditionals earlier in the readme about linux main and cross module optimizations)

Reply:
I first had that in but then I thought that it adds noise and distracts from the important stuff. More importantly though: someone who wants to learn about deploying their Swift app today is (hopefully) not using 5.1 or older anymore. People who are stuck on legacy Swifts are probably already deployed and won’t read this. WDYT?

Reply:
okay, if that is the case we should probably also remove the there pre 5.2 stuff?

Reply:
also I think we missed a spot: https://github.com/swift-server/guides#building-for-production

Reply:
done @tomerd